lib/config/fileconf.go

```diff
@@ -1013,6 +1013,10 @@ type DatabaseAWSRDS struct {
 	InstanceID string `yaml:"instance_id,omitempty"`
 	// ClusterID is the RDS cluster (Aurora) identifier.
 	ClusterID string `yaml:"cluster_id,omitempty"`
+	// ProxyName is the identifier for the RDS proxy.
+	ProxyName string `yaml:"proxy_name,omitempty"`
+	// ProxyEndpointName is the name for the RDS proxy endpoint.
+	ProxyEndpointName string `yaml:"proxy_endpoint_name,omitempty"`
```

Comment:
Do users need to set both in the config (in case of static config)?

Reply:
With the new parsing, actually none is required in config file. Same applies for ClusterID and InstanceID above